com.kassindornelles.lolforlinuxinstaller.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: com.kassindornelles.lolforlinuxinstaller
```

Comment:
`com.kassindornelles` doesn't exist. the app-id doesn't meet the requirements.

Reply:
the app is from a github repo, kassindornelles is the nome of the creator, i change it to `git.kassindornelles.lolforlinuxinstaller`?

Reply:
this is as much invalid

Reply:
see https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
is `io.github.kassindornelles.lolforlinuxinstaller` valid?